### DISCUSSION TITLE
Mealplanner creation automatically taking into account nutritional tables

### DISCUSSION THREAD

**Discussion by ch3p4ll3** [mealie-recipes/mealie#2712] [Ideas]
It would be very interesting to be able to press a button and the app automatically generates a meal plan based on the nutritional values of the dishes or the number of calories you want to ingest daily

**Comment by Belgadon** [comment]
Or some variety of meals, like:
"Create a plan for the week with 5 meals. One for each day. Witch the following restrictions: Max. two can have meat. Max. one can be Asian cuisine."

**Comment by parumpum** [comment]
This is an idea my better half was bringing up. Not difficult adding in these fields for filtering with the current meal plan editor.

My concern is that you can make a case for filtering on quite a few of these Nutrition fields, which really expands the list of fields. It's not immediately obvious that the list is even scrollable when you click on it.

<img width="250" alt="image" src="https://github.com/user-attachments/assets/76b51aed-7f8a-41df-9464-7d025ae00565" />

Feels like there may be a better way to do this:
<img width="251" alt="image" src="https://github.com/user-attachments/assets/982ff442-5fac-431c-8fb5-77bcdfbd74b5" />

Perhaps a "Nutrition" entry that expands to another entry field? Similar to how "Show Advanced" adds the parentheses fields.
